bot/cogs/defcon.py

```diff
@@ -226,18 +221,18 @@ def build_defcon_msg(self, change: str, e: Exception = None) -> str:
 
         return msg
 
-    async def send_defcon_log(self, change: str, actor: Member, e: Exception = None) -> None:
+    async def send_defcon_log(self, action: Action, actor: Member, e: Exception = None) -> None:
         """
         Send log message for DEFCON action.
 
         `change` string may be one of the following: ('enabled', 'disabled', 'updated')
```

Comment:
This docstring is no longer accurate. It can probably just be removed altogether.

Reply:
I'll update their docstring accordingly.

Reply:
Do you think it's valuable to still keep it? I feel that since it's an enum and the argument is type hinted, one can just look at the definition of the enum to see what the values can be. Before it was just `str` and that was understandably vague so this docstring made sense at the time.

Reply:
At the time, yes, since it can be any string, it is valuable to know what specific string one needs to pass into the functions. Now via type hinting I do not think we'll need to keep this anymore, we would only have one `Action` Enum now, if anything we can just add more to it like an Interface.